source/frontend/src/features/documents/documentDetail/DocumentDetailForm.tsx

```diff
@@ -92,102 +93,106 @@ export const DocumentDetailForm: React.FunctionComponent<
-                        options={documentTypeOptions}
-                        onChange={onDocumentTypeChange}
-                        disabled={
-                          documentTypeOptions.length === 1 ||
-                          props.relationshipType === ApiGen_CodeTypes_DocumentRelationType.Templates
-                        }
-                      />
-                    </SectionField>
-                    {exists(documentTypePurpose) && (
-                      <SectionField label={null}>
-                        <StyledPurposeText>{documentTypePurpose}</StyledPurposeText>
-                      </SectionField>
-                    )}
-                    <SectionField label="Status" labelWidth={{ xs: 4 }}>
-                      <Select field="documentStatusCode" options={documentStatusTypes} />
-                    </SectionField>
-                    {props.documentTypeUpdated && (
-                      <StyledDiv>
-                        <em>
-                          Some associated metadata may be lost if the document type is changed.
-                        </em>
-                      </StyledDiv>
-                    )}
-                    <StyledH3>Details</StyledH3>
-                    <DocumentMetadataView
-                      mayanMetadata={props.mayanMetadataTypes}
-                      values={formikProps.values}
-                      errors={formikProps.errors}
+        <Section
+          noPadding
+          header={
+            <>
+              Document Information
+              <TooltipIcon
+                toolTipId="documentInfoToolTip"
+                innerClassName="documentInfoToolTip"
+                toolTip="Information you provided here will be searchable"
+              />{' '}
+            </>
+          }
+        >
+          <StyledScrollable>
+            <Formik<DocumentUpdateFormData>
+              innerRef={props.formikRef}
+              initialValues={initialFormState}
+              validationSchema={DocumentUpdateFormDataYupSchema}
+              onSubmit={async (values: DocumentUpdateFormData, { setSubmitting }) => {
+                if (
+                  props.document?.pimsDocumentRelationship?.id &&
+                  values.documentStatusCode !== undefined
+                ) {
+                  const request = DocumentUpdateFormData.toRequestApi(values);
+                  await props.onUpdate(request);
+                  setSubmitting(false);
+                } else {
+                  console.error('Selected document status is not valid');
+                }
+              }}
+            >
+              {formikProps => (
+                <>
+                  {console.log(formikProps.values)}
+                  <SectionField label="File name" labelWidth={{ xs: 4 }} required>
+                    <TooltipWrapper
+                      tooltipId="file-name-tip"
+                      tooltip="Changing the document name is not supported."
```

Comment:
I'm confused by this tooltip text. Are we not letting users change the document/file name here?

Or do you mean "Changing the document/file EXTENSION is not supported" ? 

I ask because there is logic on the backend to prevent changing extensions (which makes sense)